### ISSUE TITLE
Chiaki4deck uncapable of connecting to PS4 on different network

### ISSUE THREAD

**Issue by vavasjc** [streetpea/chiaki4deck#58]
I'm using Chiaki 2.1.1. My PS4 and PS5 are with DHCP, static IP settings. Chiaki works for both PS5 and PS4 flawlessly when in same network. 

For different networks, after opening some ports in port forward and firewall at router settings, I was able to use PS5 at a different network but with a caveat: Chiaki was unable to send the wakeup message/package. I'd need to turn on my PS5 using my iPhone or notebook and then after it woke up I'd be able to connect while being in a different network.

For PS4 however, this was not possible at all. I only receive this default message:
"Chiaki Session has quit:
Connection Refused in Session Request"

I'd like to know if any of you had success or followed any guide which indeed had you succeed on Chiaki or Chiaki4deck + PS4 while in a different network

**Comment by vavasjc** [comment]
Ok guys I got it. 
So the issue is that port forwarding can only be done to a IP at a time. both PS4 Remote Play and PS5 remote play rely on ports 

- UDP 987
- UDP & TCP 9295

Because my intention was to connect to either PS5 or PS4 depending on what I wanted to play, I thought both used different ports. They don't. I'd end up needing to set beforehand to which internal IP I wanted them to forward to: To PS4 or to PS5.

Curiously, I wasn't able to connect back to my PS5 after my tests until I removed port UDP 9296 which I had just opened for PS4. It could be a coincidence but didn't seem so.

When ports 987 and 9295 are forwarding to PS5 and you try to connect to PS4 you get error "RP version is different", RP being Remote Play. This also occurs the opposite, when your ports are targeting PS4 and you try to connect to PS5

Interestingly, the official PS Remote Play from Iphone (didn't try PC or iPad but should be the same) don't care for this even when you are at a different network. PS Remote Play will work connecting to either PS4 or PS5 regardless of what the ports are forwarding to.

Maybe there's some... secret sauce Sony leverages that Chiaki can't
